### PR TITLE
7043 trigger discord thread new message #7266

### PR DESCRIPTION
**Pull request by luancazarine**
Triggers:
  - New Thread Message

### <samp>🤖 Generated by Copilot at 9ed4a24</samp>

Updated the `create-channel-invite` action to allow custom invite settings and improved code style. Incremented the version numbers of 18 other actions to indicate minor updates or bug fixes. These actions are part of the `discord_bot` component, which enables Pipedream users to interact with the Discord API.

### <samp>🤖 Generated by Copilot at 9ed4a24</samp>

> _We've patched up our actions, `0.0.8` and more_
> _We've fixed some bugs and added some features galore_
> _So grab your ropes and pull with me, on the count of three_
> _We'll sail the seas of Discord, with our actions `0.0.9` and more_

### <samp>🤖 Generated by Copilot at 9ed4a24</samp>

*  Incremented version numbers of 18 actions to reflect minor updates or bug fixes ([link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-4f0f0d43450894db36fcc55d577ea59086a3027c110f598e0e11991d8bbb5dd8L11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-9530ac41e5c606a913bed75e8aa144de517aebeda4bc592c892f73eea3161a54L11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-87fef9d9cd96fbb44fee4570addd7bea883cedb5958be437a903b965ce2d64e4L10-R10), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-6124258f6f847e1747e25768a41bc29a1dfb5c94b221439fa3c49c18f5ea1478L12-R12), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-de7b5d909e5758ee5aafd3fc8d8a220c878169121891ab07358b3a8068fced37L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-82d90ad307b6c89724fd54846aa6ea54902cf5ec0fb74f739d0be73d0857970cL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-b68f3fe0926676979787157d141aec746170086a0658e7808c1f5f8c7cfb847bL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-773edd0d826e7afd33c41009f90ca22a4fa055ebdfc3446029be757bd11759b6L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-049c9084c6399e992c96a527b9111c1ce90bc716bb9bf0995171c0d5fbbc1cafL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-1565d552327ee89d1d5b7ce026ffe26d481ce8a26f1fb015ab935fca0244a622L12-R12), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-60bc9b5438a9e2470a23f89edad92fc63afe2a0cbde8762e69f8073a5c030987L13-R13), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-aeb3053b8a4b82cbed404308533215baf82095d80bffd202bc2759a7c5fa4ca5L9-R9), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-881a821ad173a3ae8fb4582590140b70876a5a08321a7d0b682c97e1567d8f7eL14-R14), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-230a479363374dca887a7235b02acbe264f504211c966602ab79615755ebc8d8L14-R14), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-2f4ff0dc694a77ede303014cbb66768e19230948da11b09ccf2abaa4cc982391L21-R21), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-5050d6e6a4426ee09205197ed547be3263c3e1e576785938809c1833a70da5cdL22-R22), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-003b8395fc26e7ff1ee5d2a01bf9d97611ed42094a82ddc5a153004fbb8f7533L11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-f0831a3b40a476209a0a2d533c122b33e7ca1f626b028a82ba937014bb04498dL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-4c8f00cce5c5f627fe5333592675e348a0f4b107b6265f7ad617656e66dfcecfL11-R11), [link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-2a31cee3d3790d8989676646807cd889d87bea07adf71d57bc2ceea809287e4bL12-R12))
*  Removed spacing between curly braces and parentheses in `run` method of `create-channel-invite` action for consistency ([link](https://github.com/PipedreamHQ/pipedream/pull/7279/files?diff=unified&w=0#diff-87fef9d9cd96fbb44fee4570addd7bea883cedb5958be437a903b965ce2d64e4L66-R66))

### CODE REVIEW

**Review by michelle0927 — CHANGES_REQUESTED** [review]
Looks good. Just wondering if we could select which `threads` to emit events for by using a `string[]` type prop with `async options`.

File: components/discord_bot/sources/new-thread-message/new-thread-message.mjs [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import maxBy from "lodash.maxby";
+import common from "../common.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "discord_bot-new-thread-message",
+  name: "New Thread Message",
+  description: "Emit new event for each message posted to a specific thread.",
```
[the remark this answers is not in the document]
Comment (luancazarine) [reply components/discord_bot/sources/new-thread-message/new-thread-message.mjs:10]: Actually the description is wrong, the right source is for any topic thread.

**Review by michelle0927 — APPROVED** [review]
Ready for QA!

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**